catalog.js

```diff
@@ -0,0 +1,50 @@
+var product = [
+    {
+        name: 'Брюки',
+        article: 1001,
+        prise: 1499,
+
+    },
+    {
+        name: 'Майка',
+        article: 2001,
+        prise: 499,
+
+    },
+    {
+        name: 'Рубашка',
+        article: 3001,
+        prise: 799,
+
+    },
+    {
+        name: 'Ремень',
+        article: 4001,
+        prise: 299,
+    },
+]
+
+function showCatalog(product, $child) {
+    if (Object.keys(product).length !== 0) {
+        for (var i = 0; i < Object.keys(product).length; i++) {
+            var $cloneDiv = document.createElement('div');
+            $child.appendChild($cloneDiv);
+            $cloneDiv.setAttribute("id", "product_"+product[i].article);
+            var $copyP = document.createElement('p');
```

Comment:
используйте только один вид кавычек везде
```suggestion
            var $copyP = document.createElement("p");
```